Module-12/Task1/Tasks/Task7.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Task1.DoNotChange;
+
+namespace Task1.Tasks;
+
+public class Task7
+{
+    /// <summary>
+    /// Calculate the average profitability of each city (average amount of orders per customer)
+    /// and average rate (average number of orders per customer from each city). 
+    /// </summary>
+    /// <param name="customers"></param>
+    public static IEnumerable<(string city, int averageIncome, int averageIntensity)> Linq9(
```

Comment:
variable names do not look good to me, but for the purposes of this task it's ok